testgrid/config/config.yaml

```diff
@@ -1537,6 +1537,10 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-gci-new-gci-master-upgrade-cluster-new
 - name: ci-kubernetes-e2e-mlkube-gke
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-mlkube-gke
+- name: mlkube-build-presubmit
+  gcs_prefix: kubernetes-jenkins/logs/mlkube-build-presubmit
+- name: mlkube-build-postsubmit
+  gcs_prefix: kubernetes-jenkins/logs/mlkube-build-postsubmit
```

Comment:
also add to https://github.com/jlewi/test-infra/blob/8563b0c1fa76dfc890d05f38e6b9bb2faa5f76e6/testgrid/config/config.yaml#L3087-L3092